tests/performance_tests/performance_test_basic.py

```diff
@@ -43,11 +43,15 @@ class ClusterConfig():
         genesisPath: str = "tests/performance_tests/genesis.json"
         maximumP2pPerHost: int = 5000
         maximumClients: int = 0
-        loggingDict = { "bios": "off" }
+        loggingDict: dict = field(default_factory=lambda: { "bios": "off" })
+        prodsEnableTraceApi: bool = False
+        specificExtraNodeosArgs: dict = field(default_factory=dict)
         _totalNodes: int = 2
 
         def __post_init__(self):
             self._totalNodes = max(2, self.pnodes if self.totalNodes < self.pnodes else self.totalNodes)
+            if not self.prodsEnableTraceApi:
```

Comment:
Wouldn't all non-producer nodes want `trace_api_plugin` enabled, regardless of whether or not the producer nodes have it enabled?